src/NumericalAlgorithms/Spectral/SwshInterpolation.cpp

```diff
@@ -101,9 +100,8 @@ void SpinWeightedSphericalHarmonic::evaluate(
                      pow(sin_theta_over_2,
                          2 * static_cast<int>(l_) - (2 * r + spin_ - m_));
     } else if (2 * static_cast<int>(l_) < 2 * r + spin_ - m_) {
-      theta_factor = pow(cos_theta_over_2 / sin_theta_over_2,
-                         2 * r + spin_ - m_ - 2 * static_cast<int>(l_)) *
-                     pow(cos_theta_over_2, 2 * l_);
+      // the r_prefactor multiplied by theta_factor will be zero
+      theta_factor = 0.;
```

Comment:
How about `ASSERT(equal_within_roundoff(gsl::at(r_prefactors_, r), 0.), "...")` and `continue;`?